### ISSUE TITLE
pip install for Win 10 python3.7 missing two dlls

### ISSUE THREAD

**Issue by itsayellow** [wxWidgets/Phoenix#1002]
**Operating system**: Windows 10
**wxPython version & source**: wxpython 4.0.3 from pip
**Python version & source**: python 3.7 32-bit from exe installer from python.org

Installing wx via pip3 seems to be missing two dlls: msvcp140.dll and vcruntime140.dll

When I install python3.7 32-bit as the only python interpreter on my Windows system, and then install wxpython from pip3, the following occurs:
```
(virt) eviltwin:misc$ python3
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:06:47) [MSC v.1914 32 bit (Inte
l)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import wx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\mclapp\git\marcam\virt\lib\site-packages\wx\__init__.py", line
17, in <module>
    from wx.core import *
  File "C:\Users\mclapp\git\marcam\virt\lib\site-packages\wx\core.py", line 12,
in <module>
    from ._core import *
ImportError: DLL load failed: The specified module could not be found.
>>>
```

However, if I copy msvcp140.dll and vcruntime140.dll (from a different wxpython install on python 3.6.6) into my python path, everything is fine.

I notice that msvcp140.dll and vcruntime140.dll are present in a python3.6.6 virtualenv wxpython install in
```
./Lib/site-packages/wx/msvcp140.dll
./Lib/site-packages/wx/vcruntime140.dll
```
but that they are not present in a python3.7 virtualenv wxpython install.

Reference:
https://groups.google.com/d/topic/wxpython-users/j212k5MOg2c/discussion

**Comment by itsayellow** [comment]
I just tried that and can confirm that the only dll I need to manually add to my Win 10 + python3.7 + wxpython 4.0.3 environment is `msvcp140.dll`.

**Comment by RobinD42** [comment]
This has already been fixed for the 4.0.4 release. https://github.com/wxWidgets/Phoenix/blob/wxPy-4.0.x/build.py#L1373

**Comment by Mradr** [comment]
How does one install the 4.0.4 version?
